lib/std/build.zig

```diff
@@ -2075,6 +2076,9 @@ pub const LibExeObjStep = struct {
         if (self.link_eh_frame_hdr) {
             try zig_args.append("--eh-frame-hdr");
         }
+        if(self.link_emit_relocs){
```

Comment:
Run `zig fmt` on the files you modify

Reply:
Apologies - this is my first time contributing

Reply:
No problem, this is what code reviews are for :)